src/NServiceBus.Transport.RabbitMQ/Receiving/MessagePump.cs

```diff
@@ -117,7 +117,7 @@ public Task ChangeConcurrency(PushRuntimeSettings limitations, CancellationToken
         {
             maxConcurrency = limitations.MaxConcurrency;
             Logger.InfoFormat("Calling a change concurrency and reconnecting with new value {0}.", limitations.MaxConcurrency);
-            _ = Task.Run(() => Reconnect(), cancellationToken);
+            _ = Task.Run(() => Reconnect(messageProcessingCancellationTokenSource.Token), cancellationToken);
```

Comment:
I wonder why we add here the cancellation token as a second parameter to `Task.Run` this would help to make sure the task is not run in the scheduler when the token is triggered but when the task is already running it has no effect because then only the token from messageProcessingCancellationTokenSource.Token counts.

Reply:
I guess it does not harm to have it. It is very unlikely that the reconnect loop task would not ran but there is nothing in it that requires it to be invoked so its safe to pass the CT to the Task.Run